src/about/test/about.component.spec.ts

```diff
@@ -14,24 +14,29 @@ import {
 } from '../../test/';
 import { AboutComponent } from '../about.component';
 import { AboutService } from '../about.service';
+import { BehaviorSubject } from 'rxjs/BehaviorSubject';
```

Comment:
try to split import from different sources, for example:
`import from non-project files, angular, rxjs, etc`
<br>
`import what you are testing`
<br>
`imports like mocks`